docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -14,10 +14,11 @@
     "\n",
     "The content is structured as follows:\n",
     "\n",
-    "1. [Introduction](#intro)\n",
-    "2. [Classification](#class)    \n",
-    "3. [Clustering](#clustering)\n",
-    "4. [Conclusion](#conclusion)\n",
+    "1. [Introduction](#1.-Introduction)\n",
+    "2. [Classification](#2.-Classification)\n",
+    "3. [Clustering](#3.-Clustering)\n",
+    "4. [Kernel Principal Components Analysis](#4.-Kernel-Principal-Component-Analysis)\n",
+    "5. [Conclusion](#5.-Conclusion)\n",
```

Comment:
done in the last commit.

Reply:
Unfortunately the html output looks the same. You can see this if you go to Ci output and show all checks. And then click on any Details. At the top of the left pane is Summary, click that. Scroll down the page, below the mail.yml figure are Artifacts. documentation zip is a build of the docs where the notebooks are used as checked in, tutorials3.8 is similar but in this case the notebooks are executed too. Either way the output looks the same. My only thought is to perhaps move that sentence before "The content is structured as follows:" ie. before the list so that the content list is the last thing in that section. I.e after the paragraph that starts "In this notebook, you will learn how to define quantum kernel.....". I think it would be fine there right.